rdkit/Chem/Draw/MolDrawing.py

```diff
@@ -231,14 +210,15 @@ def _drawWedgedBond(self, bond, pos, nbrPos, width=None, color=None, dash=None):
 
   def _drawBond(self, bond, atom, nbr, pos, nbrPos, conf, width=None, color=None, color2=None,
                 labelSize1=None, labelSize2=None):
-    if width is None:
-      width = self.drawingOptions.bondLineWidth
-    if color is None:
-      color = self.drawingOptions.defaultColor
+    width = width or self.drawingOptions.bondLineWidth
+    color = color or self.drawingOptions.defaultColor
+    color2 = color2 or self.drawingOptions.defaultColor
     p1_raw = copy.deepcopy(pos)
     p2_raw = copy.deepcopy(nbrPos)
     newpos = self._getBondAttachmentCoordinates(p1_raw, p2_raw, labelSize1)
     newnbrPos = self._getBondAttachmentCoordinates(p2_raw, p1_raw, labelSize2)
+    addDefaultLine = functools.partial(self.canvas.addCanvasLine, linewidth=width, color=color,
+                                       color2=color2)
```

Comment:
The `self.canvas.addCanvasLine` calls in this method all have the same `linewidth`, `color`, and `color2` arguments. By using `functools.partial` we can define this here and focus on what is really different below.